### PR TITLE
Update receipies to include dependencies

### PR DESCRIPTION
**Pull request by vaelen**

### CODE REVIEW

File: gr-recipes/gr-starcoder.lwr [hunk 1]
```diff
@@ -23,7 +23,10 @@ gitbranch: master
...
 depends:
 - gr-satellites
+- libprotobuf
+- libgrpc
+- libpng
 source: git+https://github.com/infostellarinc/starcoder.git
 configuredir: gr-starcoder/build
 makedir: gr-starcoder/build
-installdir: gr-starcoder/build
+installdir: gr-starcoder/build
```
Comment (koncha99) [inline gr-recipes/gr-starcoder.lwr]: newline

File: gr-recipes/libgrpc.lwr [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+category: common
+depends: libprotobuf
+description: libgrpc
+satisfy:
+  deb: libgrpc-dev
+  rpm: grpc-devel
+gitbranch: master
+inherit: autoconf
+source: git+https://github.com/grpc/grpc.git
+gitargs: --recursive
```
Comment (koncha99) [inline gr-recipes/libgrpc.lwr]: ditto

File: gr-recipes/libpng.lwr [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+category: common
+depends:
+description: libpng
+satisfy:
+  deb: libpng-dev
+  rpm: libpng-devel
+gitbranch: master
+inherit: cmake
+source: git+https://github.com/glennrp/libpng.git
```
Comment (koncha99) [inline gr-recipes/libpng.lwr]: ditto

File: gr-recipes/libprotobuf.lwr [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+category: common
+depends:
+description: libprotobuf
+satisfy:
+  deb: libprotobuf-dev
+  rpm: protobuf-devel
+gitbranch: master
+inherit: autoconf
+source: git+https://github.com/protocolbuffers/protobuf.git
+gitargs: --recursive
```
Comment (koncha99) [inline gr-recipes/libprotobuf.lwr]: ditto

**Comment by vaelen** [comment]
This never worked right, so I will attempt to update it again in the future.

**Pull request closed without merging**